### ISSUE TITLE
buffer full error

### ISSUE THREAD

**Issue by wryanthomas** [seunomosowon/SA-geodistance#8]
I'm experiencing the following error when passing high record volume to geodistance command (in both Splunk Enterprise and Splunk Cloud):

`ERROR ChunkedExternProcessor - Failure writing result chunk, buffer full. External process possibly failed to read its stdin.`

According to another post on answers.splunk.com, it looks like someone (kulick?) might have a solution to this issue:
https://answers.splunk.com/answers/686940/how-come-custom-search-commands-csc-scpv2-cannot-h.html

He references a fix in git hub: "The solution is embodied in the new 'echo' custom command implemented in this change...
https://github.com/TiVo/splunk-sdk-python/commit/5188f7d709cadd80e786692b371a64c4ae0991d2

There is additional reporting of this issue here:
https://github.com/splunk/splunk-sdk-python/issues/150

**Comment by seunomosowon** [comment]
Hi Ryan,

The fix suggested is actually a fix to the Splunk SDK. Can you call it out to Splunk Support using your cloud entitlement? They may be able to add the fix and release a new version of the SDK, and then I can add this to the TA.

Many thanks,
Seun

**Comment by wryanthomas** [comment]
I'm looking at ...  https://github.com/splunk/splunk-sdk-python/issues/150

And in particular... https://github.com/splunk/splunk-sdk-python/commit/6b51317fb174ea6cf0d717d6cbeeb11f242fd4c5

Does this indicate that, presumably, this issue has already been resolved in latest SDK?

Or is this a separate but related issue?

**Comment by wryanthomas** [comment]
Or perhaps 150 issue was closed and handled here?
https://github.com/splunk/splunk-sdk-python/pull/301

**Comment by seunomosowon** [comment]
Yes, it was. You might be able to try it out by setting chunked to disabled
in your local commands.conf. It looks like the fix just turns off chin king
for SCPv2.

I already updated the ask early this year. I’ll check if there’s a newer
one.

Thanks again,
Seun
